views/default/page/elements/comments.php

```diff
@@ -19,22 +19,25 @@
 	$limit = elgg_trigger_plugin_hook('config', 'comments_per_page', [], 25);
 }
 
-$id = '';
-if (isset($vars['id'])) {
-	$id = "id=\"{$vars['id']}\"";
-}
+$attr = [
+	'id' => elgg_extract('id', $vars, 'elgg-comments')
+];
 
 $class = 'elgg-comments';
 if (isset($vars['class'])) {
 	$class = "$class {$vars['class']}";
 }
+$attr['class'] = $class;
 
 // work around for deprecation code in elgg_view()
 unset($vars['internalid']);
 
-echo "<div $id class=\"$class\">";
+// make the pagination go to the comments section
```

Comment:
I feel like this functionality belongs in the pagination view. Setting base_url here feels a little hacky.